Task 4

K-Shields1994 opened the pull request:
Ready for review. Implemented the Delete Rest endpoint method

K-Shields1994 committed:
Resolved all issues with InventoryController and InventoryControllerTest and changed setting in intelliJ
Reformatted code
Signed-off-by: Kevin Shields <ksjr.22@outlook.com>

Implemented the Delete Rest endpoint

Signed-off-by: Kevin Shields <ksjr.22@outlook.com>

K-Shields1994 committed:
Resolved all issues with InventoryController and InventoryControllerTest and changed setting in intelliJ
Reformatted code
Signed-off-by: Kevin Shields <ksjr.22@outlook.com>

Implemented the Delete Rest endpoint

Signed-off-by: Kevin Shields <ksjr.22@outlook.com>

JeremiahFailla commented on starter/src/test/java/com/starter/fullstack/rest/InventoryControllerTest.java:
```diff
@@ -15,7 +15,9 @@
 import org.springframework.test.context.junit4.SpringRunner;
 import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.RequestBuilder;
 
```
remove space

K-Shields1994 replied:
Removed the spaces

JeremiahFailla commented on starter/src/test/java/com/starter/fullstack/rest/InventoryControllerTest.java:
```diff
@@ -15,7 +15,9 @@
 import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.RequestBuilder;
 
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
```
I don't believe you need static in these imports

K-Shields1994 replied:
Removed the "static" from the imports

JeremiahFailla reviewed (changes requested):
Just some little feedback otherwise it looks good.

K-Shields1994 committed:
Resolved all issues with InventoryController and InventoryControllerTest and changed setting in intelliJ
Reformatted code
Signed-off-by: Kevin Shields <ksjr.22@outlook.com>

Implemented the Delete Rest endpoint

Removed spaces in the imports and removed the "static" out of the imports

Signed-off-by: Kevin Shields <ksjr.22@outlook.com>

K-Shields1994 committed:
Resolved all issues with InventoryController and InventoryControllerTest and changed setting in intelliJ
Reformatted code
Signed-off-by: Kevin Shields <ksjr.22@outlook.com>

Implemented the Delete Rest endpoint

Removed spaces in the imports and removed the "static" out of the imports

Signed-off-by: Kevin Shields <ksjr.22@outlook.com>

JeremiahFailla reviewed (approved):
Ready to merge
